assets/css/sensei-course-theme/quiz-compat.scss

```diff
@@ -15,5 +16,78 @@
 		.sensei-lms-quiz-back-to-lesson {
 			color: inherit;
 		}
+
+		input[type=file]::file-selector-button {
+			padding: 0.75rem 1.25rem;
+			color: inherit;
+		}
+	}
+}
+
+.wp-block-sensei-lms-question-answers {
+	& * {
+		font-size: 1.125rem;
+	}
+
+	input[type=file]::file-selector-button,
+	.sensei-lms-question-block__file-input-placeholder {
+		color: inherit;
+		border: 1px solid var(--sensei-secondary-color, #43AF99);
+		border-radius: 0;
+		font-size: 1.125rem;
+		text-transform: none;
+		line-height: 1;
+		font-family: inherit;
+		padding: .85rem 1.25rem;
+	}
+}
+
+.wp-block-sensei-lms-quiz {
+	.sensei-lms-question-block__title {
+		font-family: var(--wp--preset--font-family--system);
+		textarea {
+			font-size: inherit;
+			font-family: inherit;
+			font-weight: inherit;
+		}
+	}
+}
+
+.quiz {
+	form {
+		#sensei-quiz-list {
+			color: $textColor;
+
+			/* Some themes directly target inputs */
+			input {
+				color: $textColor;
+			}
+
+			li {
```

Comment:
It's worth mentioning this, although I'd be afraid to change it now 😬.

It's generally best to try to avoid element selectors when possible. The problem with using them (as opposed to classes) is that it ties the HTML structure to the CSS, so that if we were to switch from using a list to something else, all of our CSS would break and need to be updated.